### PR TITLE
add libvirt-actuator image creation and publishing

### PR DESCRIPTION
**Pull request by paulfantom**
SSIA

@smarterclayton can we also have docker hub repository created for this?

/assign @enxebre @smarterclayton

### CODE REVIEW

File: ci-operator/config/openshift/cluster-api-provider-libvirt/master.yaml [hunk 1]
```diff
@@ -6,6 +6,10 @@ base_images:
     tag: base
 binary_build_commands: make build INSTALL_DEPS=1
 canonical_go_repository: github.com/openshift/cluster-api-provider-libvirt
+images:
+- dockerfile_path: Dockerfile
+  from: base
+  to: libvirt-machine-controller
```
Comment (enxebre) [inline ci-operator/config/openshift/cluster-api-provider-libvirt/master.yaml]: we should follow same convention here https://github.com/openshift/release/blob/master/ci-operator/config/openshift/cluster-api-provider-aws/master.yaml#L12

**Comment by enxebre** [comment]
could you squash, this is only "add image creation and publishing" right? the last two commits are fixes over the first one

**Comment by paulfantom** [comment]
Done.

File: ci-operator/jobs/openshift/cluster-api-provider-libvirt/openshift-cluster-api-provider-libvirt-postsubmits-release-3.11.yaml [hunk 2]
```diff
@@ -0,0 +1,32 @@
+postsubmits:
```
[the remark this answers is not in the document]
Comment (paulfantom) [reply ci-operator/jobs/openshift/cluster-api-provider-libvirt/openshift-cluster-api-provider-libvirt-postsubmits-release-3.11.yaml]: @enxebre do we need it for 3.11 ?

[the remark this answers is not in the document]
Comment (enxebre) [reply ci-operator/jobs/openshift/cluster-api-provider-libvirt/openshift-cluster-api-provider-libvirt-postsubmits-release-3.11.yaml]: no. we are targeting 4.0

**Comment by paulfantom** [comment]
@smarterclayton 3.11 tests are removed in #1484. Can we have an image in dockerhub now?

**Pull request merged**